client/verta/verta/deployment/_deployedmodel.py

```diff
@@ -57,60 +55,25 @@ class DeployedModel:
         # <DeployedModel 01234567-0123-0123-0123-012345678901>
 
     """
-    def __init__(self, host=None, run_id=None, creds=None, access_token=None, prediction_url=None, **kwargs):
+    def __init__(self, prediction_url, token=None, creds=None):
         # TODO: put automodule verta.deployment back on ReadTheDocs
+        self.prediction_url = prediction_url
         creds = creds or credentials.load_from_os_env()
-        self._session = self._make_session(creds, access_token)
-
-        # TODO: remove the following block of param checks
-        # this is to temporarily maintain compatibility with anyone passing in `socket` and `model_id` as kwargs
-        self._host = self._validate_host(host, kwargs.get('socket'))
-        self._id = self._validate_id(run_id, kwargs.get('model_id'))
-
-        self._prediction_url = prediction_url
-        self._validate_run_id_or_prediction_url()
-
-        back_end_url = urlparse(host)
-        self._socket = back_end_url.netloc + back_end_url.path.rstrip('/')
-        self._scheme = back_end_url.scheme or ("https" if ".verta.ai" in self._socket else "http")
+        self._credentials = creds
+        self._access_token = token
+        self._session = self._make_session()
 
-
-    def _make_session(self, credentials, access_token):
+    def _make_session(self):
```

Comment:
Would you know if we should call `close()` on the prior session if there is one?

Reply:
I'm not sure but I've changed `_make_session` to `_init_session` to handle closing any previously existing session.